canonical/jp.ubuntu.com
# Refresh /pro

## Done

- Apply page refresh on `/enterprise-support`
- Change all occurrences of `/enterprise-support` to `/pro`

## QA

- Go to https://jp-ubuntu-com-489.demos.haus/pro
- Check that it matches [design](https://www.figma.com/design/tI8YK4wQnKiWLAxdX1fWnB/24.10-jp.ubuntu.com?node-id=311-4080&m=dev) and [docs](https://docs.google.com/document/d/1xmk7zYUzWILnbW3RxGtC9c9stYYEFtKBytETlIgyHYE/edit)

## Issue / Card

Fixes [WD-11642](https://warthogs.atlassian.net/browse/WD-11642)

## Screenshots

[if relevant, include a screenshot]


[WD-11642]: https://warthogs.atlassian.net/browse/WD-11642?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
